# German (Germany) translations for fb_vmware.
# Copyright (C) 2022 Frank Brehm, Berlin
# This file is distributed under the same license as the fb_vmware project.
# Frank Brehm <frank@brehm-online.com>, 2022.
#
msgid ""
msgstr ""
"Project-Id-Version: fb_vmware 0.2.4\n"
"Report-Msgid-Bugs-To: frank@brehm-online.com\n"
"POT-Creation-Date: 2022-05-02 09:46+0200\n"
"PO-Revision-Date: 2022-05-02 09:50+0200\n"
"Last-Translator: Frank Brehm <frank@brehm-online.com>\n"
"Language: de_DE\n"
"Language-Team: Frank Brehm <frank@brehm-online.com>\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.10.1\n"

#: lib/fb_vmware/about.py:282 lib/fb_vmware/cluster.py:316 lib/fb_vmware/dc.py:160
msgid "The given parameter {p!r} on calling method {m}() has failing attributes"
msgstr "Dem übergebenen Parameter {p!r} beim Aufruf der Methode {m}() fehlen Attribute"

#: lib/fb_vmware/about.py:290 lib/fb_vmware/cluster.py:324 lib/fb_vmware/dc.py:168
msgid "Parameter {t!r} must be a {e} object, a {v} object was given instead."
msgstr "Der Parameter {t!r} muss ein {e}-Objekt sein, statt dessen wurde ein {v}-Objekt übergeben."

#: lib/fb_vmware/about.py:303 lib/fb_vmware/cluster.py:349 lib/fb_vmware/dc.py:188
msgid "Creating {} object from:"
msgstr "Erstelle ein {}-Objekt aus:"

#: lib/fb_vmware/about.py:339
msgid "Created {} object:"
msgstr "Erstelltes {}-Objekt:"

#: lib/fb_vmware/base.py:106
msgid "Invalid port number {!r} for the VSphere server, must be 0 < PORT <= 65536."
msgstr ""
"Ungültige Portnummer {!r} für den vSphere-Server, diese muss größer als Null und kleiner oder "
"gleich 65536 sein."

#: lib/fb_vmware/base.py:221
msgid "Connecting to vSphere host {h}:{p} as {u!r} ..."
msgstr "Verbinde mich mit vSphere-Host {h}:{p} als {u!r} …"

#: lib/fb_vmware/base.py:239
msgid "Disconnecting from vSphere host {}."
msgstr "Trenne Verbindung zu vSphere-Host {}."

#: lib/fb_vmware/cluster.py:263 lib/fb_vmware/dc.py:126
msgid "Comparing {} objects ..."
msgstr "Vergleiche {}-Objekte …"

#: lib/fb_vmware/cluster.py:299 lib/fb_vmware/cluster.py:306
#, fuzzy
msgid "The given parameter {p!r} on calling method {m}() is not a sequence type."
msgstr "Der übergebene Parameter {p!r} beim Aufruf der Methode {m}() ist kein Sequenz-Typ."

#: lib/fb_vmware/cluster.py:357
msgid "Cluster {c!r} has network {n!r}."
msgstr "Das Cluster {c!r} besitzt das Netzwerk {n!r}."

#: lib/fb_vmware/cluster.py:364
msgid "Cluster {c!r} has datastore {d!r}."
msgstr "as Cluster {c!r} besitzt den Datastore {d!r}."

#: lib/fb_vmware/errors.py:38
msgid "No VSphere datastores found."
msgstr "Keine VSphere-Datastores gefunden."

#: lib/fb_vmware/errors.py:68
msgid "Invalid name {n!r} for a {o} VSphere object."
msgstr "Ungültiger Name für ein {o}-VSphere-Objekt."

#: lib/fb_vmware/errors.py:71
msgid "Invalid name {!r} for a VSphere object."
msgstr "Ungültiger Name für ein VSphere-Objekt."

#: lib/fb_vmware/errors.py:89
msgid "The VSphere datacenter {!r} is not existing."
msgstr "Das VSphere-Datacenter {!r} existiert nicht."

#: lib/fb_vmware/errors.py:105
msgid "The VSphere Virtual machine {!r} was not found."
msgstr "Die VSphere-VM {!r} wurde nicht gefunden."

#: lib/fb_vmware/errors.py:125
msgid "No SAN based datastore found with at least {m:0.0f} MiB == {g:0.1f} GiB available space found."
msgstr ""
"Kein SAN-basierter Datastore mit wenigstens {m:0.0f} MiB == {g:0.1f} GiB verfügbarem Platz "
"gefunden."

#: lib/fb_vmware/errors.py:143
msgid "The network {!r} is not existing."
msgstr "Das Netzwerk {!r} existiert nicht."

#: lib/fb_vmware/errors.py:162
msgid "Could not connect to the vSphere host {h}:{p} as user {u!r}."
msgstr "Konnte mich nicht zum vSphere-Host {h}:{p} als Nutzer {u!r} verbinden."

#: lib/fb_vmware/errors.py:184
msgid "Timeout on creating VM {vm!r} after {to:0.1f} seconds."
msgstr "Zeitüberschreitung bei der Erstellung der VM {vm!r} nach {to:0.1f} Sekunden."

#: lib/fb_vmware/errors.py:187
msgid "Timeout on creating VM {!r}."
msgstr "Zeitüberschreitung bei der Erstellung der VM {!r}."

#: lib/fb_vmware/obj.py:77
msgid "The type of a {} may not be None."
msgstr "Der Typ von einem {} darf nicht None sein."

#: lib/fb_vmware/obj.py:82 lib/fb_vmware/obj.py:105 lib/fb_vmware/obj.py:128
msgid "Invalid {w}.{p} {v!r}."
msgstr "{w}.{p} ist ungültig: {v!r}."

#: lib/fb_vmware/obj.py:144
msgid "The name prefix of a {} may not be None."
msgstr "Der Namenspräfix eines {} darf nicht None sein."

#: lib/fb_vmware/obj.py:148
msgid "Invalid name prefix {p!r} for a {o}."
msgstr "Ungültiger Namenspräfix {p!r} für ein {o}."

#: lib/fb_vmware/xlate.py:98
msgid "Module directory:"
msgstr "Modul-Verzeichnis:"

#: lib/fb_vmware/xlate.py:99
msgid "Base directory:"
msgstr "Basis-Verzeichnis:"

#: lib/fb_vmware/xlate.py:100
msgid "Locale directory:"
msgstr "Verzeichnis für Locales:"

#: lib/fb_vmware/xlate.py:101
msgid "Locale domain:"
msgstr "Locale-Domäne:"

#: lib/fb_vmware/xlate.py:102
msgid "Found .mo-file:"
msgstr "Gefundene .mo-Datei:"

